Docker compose error

Docker compose fails on building the mellivora image, the error being "configure: error: Package requirements (oniguruma) were not met:".

As noted in https://github.com/docker-library/php/issues/880 adding libonig-dev to apt install solves the issue (I did not removed mbstring as they say though).

The full error message is:

```
checking for oniguruma... no
configure: error: Package requirements (oniguruma) were not met:

No package 'oniguruma' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables ONIG_CFLAGS
and ONIG_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
ERROR: Service 'mellivora' failed to build: The command '/bin/sh -c docker-php-ext-install mbstring curl pdo pdo_mysql' returned a non-zero code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,7 +8,8 @@ RUN apt-get update && apt-get install -y \
     libcurl4-openssl-dev \
     pkg-config \
     zip \
-    unzip
+    unzip \
+    libonig-dev
 
 RUN docker-php-ext-install mbstring curl pdo pdo_mysql
 

